rust_src/src/multibyte.rs

```diff
@@ -87,6 +87,14 @@ impl LispStringRef {
         self.data as *mut c_char
     }
 
+    pub fn const_data_ptr(&self) -> *const c_uchar {
```

Comment:
These don't add much value, I'd just cast to `*const` where needed. Or can't you anyway pass a `*mut T` where a `*const T` is requested?